jdbc/src/main/java/com/yahoo/ycsb/db/JdbcDBClient.java

```diff
@@ -474,7 +486,21 @@ public Status insert(String tableName, String key, HashMap<String, ByteIterator>
       for (String value: fieldInfo.getFieldValues()) {
         insertStatement.setString(index++, value);
       }
-      int result = insertStatement.executeUpdate();
+      int result;
+      if (batchSize > 0) {
+        insertStatement.addBatch();
+        if (++numRowsInBatch % batchSize == 0) {
+          int[] results = insertStatement.executeBatch();
+          for (int r : results) {
+            if (r != 1) {
+              return Status.ERROR;
+            }
+          }
+        }
+        return Status.OK;
```

Comment:
Thanks for testing. 
Added batch execution for partial batches in cleanup. I think in almost all of the cases, we will have num rows to be a multiple of batch size, but agreed that it is better to be safe. 
numRowsInBatch is a long. No way we are inserting 2^63 rows from a single thread. 
I've added the BATCHED_OK to be a generic Status. Seems useful across all bindings.